api/external/applications/applications.proto

```diff
@@ -67,6 +67,17 @@ service ApplicationsService {
     option (chef.automate.api.iam.policy).action = "applications:serviceGroups:list";
   };
 
+  rpc DeleteDisconnectedServices(DisconnectedServicesReq) returns (ServicesRes ) {
+    option (google.api.http) = {
+      post: "/beta/applications/delete_disconnected_services"
```

Comment:
Any reason why we didn’t do a DELETE request instead of POST?

Reply:
For context: https://github.com/chef/automate/pull/852#discussion_r302424709

Reply:
I haven't re-read the spec recently, but according to my memory a DELETE request is supposed to refer to one specific resource in a REST context, like if we had something like `DELETE /services/:service_id` that would be correct. Since we are sending a command to delete anything that matches a specification, it's more of an RPC that should be a POST.

Reply:
Yeah the RFC says:

> The DELETE method requests that the origin server delete the resource
   identified by the Request-URI.

in this case there is not any specific resource identified by this URI.

cf. https://tools.ietf.org/html/rfc2616#section-9.7

Reply:
Makes sense!